### PR TITLE
Add e2e test

### PR DESCRIPTION
**Pull request by waprin**
Also add missing license header

### CODE REVIEW

File: e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# E2E Test for this sample app
```
Comment (jonparrott) [inline e2e/README.md]: Hey @waprin, how about a makefile to do this for us?

File: e2e/test_e2e.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    assert b'Guestbook' in r.content
+    u = uuid.uuid4()
+    data = {'content': str(u)}
+    r = requests.post(URL + '/sign', data)
+    assert r.status_code == 200
+    r = requests.get(URL)
+    assert str(u).encode('utf-8') in r.content
+    print("Success")
+
+if __name__ == "__main__":
```
Comment (jonparrott) [inline e2e/test_e2e.py]: If you just run this with pytest you'll get reporting and you won't need this.

File: e2e/README.md (hunk 1 above)
Comment (waprin) [inline e2e/README.md]: still need to set the URL themselves..a Makefile to run pip install + a script seems a bit overkill to me.

File: e2e/test_e2e.py (hunk 2 above)
Comment (waprin) [inline e2e/test_e2e.py]: yes I'm aware..also seems a bit overkill to me to add pytest dependency to run a single script.

File: e2e/README.md (hunk 1 above)
Comment (waprin) [inline e2e/README.md]: actually I guess if you include the deploy step it's not bad, I'll give that a shot

Comment (jonparrott) [inline e2e/README.md]: Yeah include the deploy, see: https://github.com/GoogleCloudPlatform/python-compat-runtime/blob/master/tests/e2e/app/Makefile

File: e2e/test_e2e.py (hunk 2 above)
Comment (jonparrott) [inline e2e/test_e2e.py]: *insert shruggie here*

Comment (jonparrott) [inline e2e/test_e2e.py]: That is to say; use your best judgement. I'm fine with whatever you decide.

Comment (waprin) [inline e2e/test_e2e.py]: yeah planning to keep it, generally see your point that these main runners clutter up your tests when you're running a bunch of tests with pytest, but for just this one script voting to keep two small lines to remove a dependency.

**Comment by waprin** [comment]
added makefile

**Pull request merged**